docs/src/getting-started/contracts.md

```diff
@@ -24,7 +20,7 @@ You can then use the contract methods very simply:
 {{#include ../../../examples/contracts/src/lib.rs:use_deployed_contract}}
 ```
 
-Alternatively, you can use `DeployConfiguration` to configure the deployment. You can, for example, deploy multiple instances of the same contract, change the storage or `tx_parameters` or update the binary using `configurables`.
+Alternatively, you can use `LoadConfiguration` to configure how the contract is loaded. You can, for example, load the same contract with different `Salt`, change the contract's storage or update the binary using `configurables`.
```

Comment:
I'm unsure if we should mention what salt does somewhere or if we rely on upstream docs to convey this. i.e. it leads to a different contract-id being computed, enabling you to deploy the same contract multiple times.

Reply:
Good point. I will mention it here